docs/tutorials/gluon/model_parallelism.md

```diff
@@ -0,0 +1,373 @@
+
+# Model parallelism
```

Comment:
you haven't updated the `tutorials/index.md` 
please also update the nightly/tests/test_tutorial_config.txt` and run `python test_tutorial.py --tutorial gluon_model_parallelism` to make sure your notebook runs

Reply:
Thanks. I'll do this.